sunpy/net/helioviewer.py

```diff
@@ -102,16 +145,16 @@ def download_jp2(self, date, directory=None, overwrite=False, **kwargs):
         date : `datetime.datetime`, string
             A string or datetime object for the desired date of the image
         directory : string
-            (Optional) Directory to download JPEG 2000 image to.
+            Directory to download JPEG 2000 image to.
         observatory : string
-            (Optional) Observatory name
+            Observatory name
         instrument : string
-            (Optional) instrument name
+            instrument name
         detector : string
-            (Optional) detector name
+            detector name
         measurement : string
-            (Optional) measurement name
-        sourceId : int
+            measurement name
+        sourceid : int
             (Optional) data source id
         jpip : bool
             (Optional) Returns a JPIP URI if set to True
```

Comment:
Do we want to support Jpip? We could remove this.

Reply:
No, we don't support it. I was thinking to remove it but then I thought I will remove when you approve it

Reply:
Remove it and the others can decide when they review the code.

Reply:
Ok, I will remove it right away.